framework/molecules/context-navs/_preview.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
Correct me if I'm wrong but you don't need this preview, the default template is enough.